Sources/IO/Geometry/GLTFImporter/ORMTexture.worker.js

```diff
@@ -0,0 +1,37 @@
+import registerWebworker from 'webworker-promise/lib/register';
+
+/**
+ *
+ * @param {ArrayBuffer} imageBuffer
+ * @param {string} mimeType
+ * @param {string} channel
+ * @returns {Promise<ImageData>}
+ */
+registerWebworker(async ({ imageBuffer, mimeType, channel }) => {
+  console.time('Texture generation');
+
+  const blob = new Blob([imageBuffer], { type: mimeType });
+  const img = await createImageBitmap(blob);
+  const canvas = new OffscreenCanvas(img.width, img.height);
+  const ctx = canvas.getContext('2d');
+
+  ctx.drawImage(img, 0, 0, img.width, img.height);
+  const bitmap = ctx.getImageData(0, 0, img.width, img.height);
+
+  if (channel) {
+    let idx;
+    if (channel === 'r') idx = 0;
+    if (channel === 'g') idx = 1;
```

Comment:
else if

Reply:
or use a const to map  str to int {"r": 0, "g": 1 ...}